yt/yt/ytlib/chunk_client/multi_reader_manager_base.cpp

```diff
@@ -191,11 +187,6 @@ void TMultiReaderManagerBase::OnReaderCreated(
         return;
     }
 
-    {
-        auto guard = Guard(PrefetchLock_);
-        CreatingReader_ = false;
```

Comment:
It seems that correct solution is to set here `true`

Reply:
I agree.

Instead of dropping this (clearly invalid) code I suggest we try to fix it.